CMakeLists.txt

```diff
@@ -192,6 +192,7 @@ add_cxx_compiler_option ("-Wall")
 add_cxx_compiler_option ("-Wextra")
 add_cxx_compiler_option ("-Wno-overloaded-virtual")
 add_cxx_compiler_option ("-Wno-deprecated")
+# add_cxx_compiler_option ("-fsanitize=null")
```

Comment:
Why this line?

Reply:
This is what makes this program crash.
@fruffy made a similar change in his testing code that will run on CI, but this is a reminder how it can be done when running normal tests too. We may consider adding this always as well (I haven't measured the cost.)

Reply:
Ahh, makes sense. Thanks.